application/src/main/resources/ecc.yml

```diff
@@ -19,12 +19,24 @@
 ## Properties for connection to Cassandra nodes
 ##
 connection:
+  # Configuration for the ThreadPoolTaskExecutor used by ecChronos.
+  # This thread pool controls how many concurrent threads the NodeManager can use.
+  # The number of threads should be proportional to the number of nodes managed by this instance.
+  threadPool:
+    ## Minimum number of threads in the pool
+    corePoolSize: 4
+    ## Maximum number of threads in the pool
+    maxPoolSize: 10
+    ## Queue capacity for pending tasks
+    queueCapacity: 20
+    ## Keep idle threads alive for 60 seconds
```

Comment:
Rephrase, e.g. "The time to keep idle threads alive, in seconds" ... no point in explicitly say "60", unless you want to state the default value